### ISSUE TITLE
Git logs commit

### ISSUE THREAD

**Issue by shaidar** [mitodl/edx-platform#205]
Below is the single git logs commit in `mitx/juniper` that needs to be submitted on the `mitx/koa` branch:
```
78901d9e5aa3698f91e961760cd706a75e6959e7	Adding error catching for long git logs
```

### acceptance criteria

- [x] PR merged into [xpro/koa branch](https://github.com/mitodl/edx-platform/tree/xpro/koa)
- [x] PR merged into [mitx/koa branch](https://github.com/mitodl/edx-platform/tree/mitx/koa)

**Comment by pdpinch** [comment]
This can definitely be included in #197

**Comment by HamzaIbnFarooq** [comment]
Issue solved by https://github.com/mitodl/edx-platform/pull/220 and https://github.com/mitodl/edx-platform/pull/221
